manifests/params.pp

```diff
@@ -38,7 +38,7 @@
       ]
     }
     'LinuxMint': {
-      $origins = ['${distro_id}:${distro_codename}-security',] #lint:ignore:single_quote_string_with_variables
+      $origins = ['origin=${distro_id}:${distro_codename}-security',] #lint:ignore:single_quote_string_with_variables
```

Comment:
I don't use mint, but I wonder if this should be
```suggestion
      $origins = ['origin=${distro_id},suite=${distro_codename}-security',] #lint:ignore:single_quote_string_with_variables
```

Reply:
Thanks! I'm not familiar enough with Mint to know the correct origin line, but since Mint is based on Ubuntu I think it makes sense to have this same format.

I can confirm the `unattended-upgrade` command runs without errors using your suggestion (although I don't have any pending updates to confirm check things work as they should - I'll monitor).